daemon/api_users_test.go

```diff
@@ -946,6 +947,76 @@ func (s *userSuite) TestPostCreateUserAutomaticDisabled(c *check.C) {
 	c.Check(users, check.HasLen, 0)
 }
 
+func (s *userSuite) TestPostCreateUserExpirationHappy(c *check.C) {
+	expectedUsername := "karl"
+	expectedEmail := "popper@lse.ac.uk"
+	// strip away subsecond values which are lost during json marshal/unmarshalling
+	expectedTime := time.Now().Add(time.Hour * 24).Round(time.Second)
```

Comment:
(just curious) any reason to use Round() vs "Truancate()" ?

Reply:
According to various sources `Truncate` can behave differently if the timestamp is not UTC. This can cause unexpected behaviour with the time.